visual-js/visual-snapshots/test/app/pdf-converter.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { PdfConverter } from "../../src/app/pdf-converter.js";
+
+jest.mock("pdf-to-img", () => {
+  const asyncIterable = {
+    [Symbol.asyncIterator]() {
+      return {
+        i: 0,
+        next() {
+          if (this.i < 2) {
+            return Promise.resolve({
+              value: `fake-image-buffer-${this.i++}`,
+              done: false,
+            });
+          }
+
+          return Promise.resolve({ done: true });
+        },
+      };
+    },
+  };
```

Comment:
Won't writing a generator work here?
```suggestion
async function* asyncIterable() {
  for (let i=0; i<2; i++) {
    yield `fake-image-buffer-${i}`;
  }
}
```